pages/index.jsx

```diff
@@ -11,10 +11,12 @@ export default function Home() {
 		<>
 			<Header />
 			<main className={styles.main}>
-				<LinkCard name='Asks' url='/ask' icon={<AsksIcon />} />
-				<LinkCard name='Statistics' url='/statistics' icon={<StatisticsIcon />} />
-				<LinkCard name='My Profile' url='/myProfile' icon={<MyProfileIcon />} />
-				<LinkCard name='Settings' url='/settings' icon={<SettingsIcon />} />
+				<div className={styles.cards}>
```

Comment:
`cards` is not a good name for the identification. They links are cards style wise, but before that they are used functionally as links - `pages-links` would fit better

Reply:
noted